BackEndApp/authv1/emails.py

```diff
@@ -0,0 +1,28 @@
+from django.core.mail import send_mail
+from BackEndApp.settings import EMAIL_HOST_USER
+
+
+class Email_Templates:
+    def __init__(self, user, request):
+        self.user = user
+        self.request = request
+
+    def update_password(self, username, email, otp):
+        subject = "Auto-DL Request to Update Password"
+        msg = (
+            "You ("
+            + username
+            + ") requested to update your password. Click here to reset your password "
```

Comment:
We are not doing the link right. Please change the messgae

Reply:
yesss just edited that....please check